eclipse-m2e/pom.xml

```diff
@@ -10,6 +10,7 @@
   </parent>
   <artifactId>devonfw-ide-eclipse-m2e</artifactId>
   <packaging>jar</packaging>
+  <version>dev-SNAPSHOT</version>
```

Comment:
this is redundant, you can omit it, but it also does not harm.
But if this SNAPSHOT is preventing the automated closing and releasing in nexus, please remove this module from parent `pom.xml` in `<modules>` section.

Reply:
You can even remove the module in `deploy` profile by copying the `<modules>` section to `deploy` profile in ide parent POM but without `eclipse-m2e`.